bin/add-font-assets.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+
+const fs = require('fs');
+
+const json = fs.readFileSync('package.json', 'utf8');
+
+const pack = JSON.parse(json);
+
+if (!pack.rnpm) {
+  pack.rnpm = {
+    assets: [],
+  };
+} else if (!pack.rnpm.assets) {
+  pack.rnpm.assets = [];
+}
+
+for (let i = 0; i < pack.rnpm.assets.length; i += 1)
+  if (pack.rnpm.assets[i] === './assets/fonts') process.exit();
+
+pack.rnpm.assets.push('./assets/fonts');
+
+fs.writeFileSync('package.json', JSON.stringify(pack, null, 2), 'utf8');
```

Comment:
Very small detail, but most prefer files ending with a new line and many editors will add it by just saving. To avoid diffs, could you add one?

Reply:
I do too, I don't know if it's git or my editor (currently JetBrains) but I have to have two blank lines at the end to make one blank line in the saved file.

Reply:
I meant in the generated package.json

Reply:
Ah, got it!